app/Http/Controllers/Api/ApplicationsController.php

```
@@ -1093,6 +1097,28 @@ private function create_application(Request $request, $type)
+                    $config = loadConfigFromGit(
+                        $gitUrl,
+                        $application->git_branch,
+                        $application->base_directory ?? '/',
+                        $destination->server->id,
+                        $teamId
+                    );
+                    if ($config) {
+                        $application->setConfig($config, fromRepository: true);
+                    }
+                } catch (\Exception $e) {
+                    \Log::warning('coolify.json: Failed to apply config via API - '.$e->getMessage());
```

PHPMD (2.15.0)

Avoid using static access to class '\Log' in method 'create_application'. (undefined)

(StaticAccess)